packages/quantic/force-app/main/default/lwc/quanticSvg/quanticSvg.js

```diff
@@ -0,0 +1,23 @@
+import { LightningElement, api } from 'lwc';
+// @ts-ignore
+import pageNotFound from './pageNotFound.html';
+// @ts-ignore
+import desert from './desert.html';
+// @ts-ignore
+import quanticSvg from './quanticSvg.html';
+
+export default class QuanticSvgLoader extends LightningElement {
+  /** @type {string} */
+  @api name;
+
+  render() {
+    switch (this.name) {
+      case "pageNotFound":
+        return pageNotFound;
+      case "NoResults":
+        return desert;
```

Comment:
I'd suggest giving a more meaningful name to the template, something like `noResults.html`.